stock_prices/stock_prices.py

```diff
@@ -3,7 +3,9 @@
 import argparse
 
```

Comment:
Very nice! simple, clever solution. Here's a visualization of your solution [https://goo.gl/F389NK](url)